src/dataview64.js

```diff
@@ -10,10 +10,24 @@ export default class DataView64 {
   getUint64(offset, littleEndian) {
     const left = this.getUint32(offset, littleEndian);
     const right = this.getUint32(offset + 4, littleEndian);
+    let combined;
     if (littleEndian) {
-      return (left << 32) | right;
+      combined = left + 2 ** 32 * right;
```

Comment:
I'm a little confused, it looks like the left/right semantic was flipped here, right? Also, as far as I know `x << 32` is essentially the same as `2 ** 32 * x` or is this different in JS?

Reply:
Could you clarify what you mean by the "semantic"?  This seems to be working for me.

Reply:
Also I don't think the syntaxes are the same since those only convert 32 bit numbers.